dags.yaml

```diff
@@ -281,7 +281,7 @@ bqetl_internet_outages:
   default_args:
     owner: aplacitelli@mozilla.com
     start_date: "2020-01-01"
-    email: ["aplacitelli@mozilla.com", "sguha@mozilla.com"]
+    email: ["aplacitelli@mozilla.com"]
```

Comment:
@Dexterp37 Does it make sense for you to be listed as sole owner for internet outages datasets now?

Reply:
Yes, it does